lerobot-web-be/app/routes/ai_control.py

```
@@ -0,0 +1,32 @@
+@router.post("/ai-control/stop", response_model=AIControlResponse, tags=["control"])
+async def ai_control_stop():
+    result = await ai_control_manager.stop()
+    return AIControlResponse(**result)
+
+@router.post("/user-models", response_model=UserModelsResponse, tags=["status"])
+def list_user_models(req: UserModelsRequest):
+    try:
+        hf_api = HfApi(token=req.api_key)
+        models_list = hf_api.list_models(author=req.user_id)
+        models = [{"modelId": m.modelId, "id": m._id, "private": m.private, "createdAt": m.created_at} for m in models_list]
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)